cookbook_se/Dockerfile.sagemaker

```diff
@@ -16,31 +15,25 @@ RUN : \
     && apt-get install -y python3.8 python3-pip python3-venv make build-essential libssl-dev curl vim
 
 # Install the AWS cli separately to prevent issues with boto being written over
-RUN pip3 install awscli
+RUN pip install awscli
 
+ENV VENV /opt/venv
 # Virtual environment
-RUN python3.8 -m venv ${VENV}
-RUN ${VENV}/bin/pip install wheel
+RUN python3 -m venv ${VENV}
+ENV PATH="${VENV}/bin:$PATH"
 
 # Install Python dependencies
 COPY ./env/sagemaker/requirements.txt /root
-RUN ${VENV}/bin/pip install -r /root/requirements.txt
+RUN pip install -r /root/requirements.txt
 
 # Setup Sagemaker entrypoints
 ENV SAGEMAKER_PROGRAM /opt/venv/bin/flytekit_sagemaker_runner.py
-RUN pip3 install --upgrade sagemaker-training==3.6.2 natsort
+RUN pip install --upgrade sagemaker-training==3.6.2 natsort
 
 # Copy the actual code
 COPY . /root
 
-# Copy over the helper script that the SDK relies on
-RUN cp ${VENV}/bin/flytekit_venv /usr/local/bin/
-RUN chmod a+x /usr/local/bin/flytekit_venv
-
 # This tag is supplied by the build script and will be used to determine the version
 # when registering tasks, workflows, and launch plans
 ARG tag
 ENV FLYTE_INTERNAL_IMAGE $tag
```

Comment:
might need to add `ENTRYPOINT []` to this dockerfile as well.

Reply:
no need of entrypoint - why do you say so. I have added the venv to PATH and that will auto find the file